plaso/parsers/dsv_parser.py

```diff
@@ -114,6 +118,63 @@ def _CreateDictReader(self, line_reader):
         quotechar=quotechar, restkey=magic_test_string,
         restval=magic_test_string)
 
+  # pylint: disable=missing-return-type-doc
+  def _CreateLineReader(self, file_object):
+    """Returns an object that returns lines from a text file.
+
+    The line reader is advanced to the beginning of the DSV content, skipping
+    any header lines.
+
+    Args:
+      file_object (dfvfs.FileIO): file-like object.
+
+    Returns:
+      TextFile|BinaryLineReader: an object that implements an iterator
+          over lines in a text file.
+
+    Raises:
+      UnicodeDecodeError: if the file cannot be read with the specified
+          encoding.
+    """
+    # The Python 2 csv module reads bytes and the Python 3 csv module Unicode
+    # reads strings.
+    if py2to3.PY_3:
+      line_reader = text_file.TextFile(file_object, encoding=self._encoding)
+    else:
+      line_reader = line_reader_file.BinaryLineReader(file_object)
+    # If we specifically define a number of lines we should skip, do that here.
+    for _ in range(0, self.NUMBER_OF_HEADER_LINES):
```

Comment:
Can you have multiple header lines? Or is it "header is present or not" ?
if the latter, maybe have boolean switch instead?

Reply:
The idea behind this seems to have been that there might be multiple header lines (I suspect like the iis format has). As far as I can tell, this constant is never actually set in any subclass, and can be removed. I'd prefer to do that in a different PR though, just in case someone does need this.